camel-cdi-integration/src/main/java/io/silverware/microservices/providers/camel/CamelCdiContextFactory.java

```diff
@@ -62,8 +61,7 @@ public CamelContext createCamelContext(final Context context) {
          if (cdiSilverService.isDeployed()) {
             log.info("CDI SilverService connected successfully!");
 
-            CamelContextMap camelContextMap = cdiSilverService.findByType(CamelContextMap.class);
-            return camelContextMap.getCamelContext("camel-1");
+            return cdiSilverService.findByType(CamelContext.class);
```

Comment:
The Camel CDI component has changed in the latest version so this change is necessary. However, I do not know why we used `camel-1` context before. I think it was just randomly chose name and the same context is always used in SilverWare.